.github/config/out_of_tree_extensions.cmake

```diff
@@ -74,7 +74,7 @@ duckdb_extension_load(excel
     APPLY_PATCHES
     LOAD_TESTS
     GIT_URL https://github.com/duckdb/duckdb-excel
-    GIT_TAG 7e97933214d0c7de2315668ec68589ae85651afb
+    GIT_TAG cf00672f2d16685d9aefcca48c6a04d8c37d7015
```

Comment:
Are we intentionally going to a different commit here?

Reply:
This is also marginal, expecially since it needs to be reworked/reviewed completely.
But the diff: https://github.com/duckdb/duckdb-excel/compare/7e97933214d0c7de2315668ec68589ae85651afb...cf00672f2d16685d9aefcca48c6a04d8c37d7015 is not on the code side, so it's irrelevant.